rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -2413,4 +2413,41 @@ JobValidationData validateJobSignal(@HeaderParam("sessionid") String sessionId,
     @Produces(MediaType.APPLICATION_JSON)
     Map<String, Map<String, Boolean>> checkJobsPermissionMethods(@HeaderParam("sessionid") String sessionId,
             PermissionForm multipart) throws RestException;
+
+    /**
+     *
+     * Add an external endpoint url to the endpoint list of the considered job
+     *
+     * @param sessionId
+     *            current session
+     * @param jobId
+     *            id of the job
+     * @param externalEndpointUrl
+     *          the external endpoint url to add to the job endpoint list
+     */
+    @POST
+    @Path("job/{jobid}/externalendpointurls")
```

Comment:
I would prefer
"jobs/{jobid}/externalendpoint"

there are several method with job/{jobid}... but it's a mistake (do not fix other methods in this PR please)

Reply:
I will update it with  jobs/{jobid}/externalendpointurl